lms/templates/learner_dashboard/programs.html

```diff
@@ -20,6 +20,16 @@
 <%block name="pagetitle">${_("Programs")}</%block>
 
 <div class="program-list-wrapper">
+    % if not len(programs):
+        <div class="dashboard">
+        <section class="empty-dashboard-message">
+            <p>${_("It looks like you aren't yet enrolled in any XSeries Programs!")}</p>
+              <a class="find-xseries-programs" href=""><i class="action-xseries-icon" aria-hidden="true"></i>
```

Comment:
@tasawernawaz I think the thing to do is similar to what's being done here https://github.com/edx/edx-platform/blob/393e885e4b6e701968150db09b87c93d227edb95/lms/djangoapps/learner_dashboard/tests/test_programs.py#L66-L67 - concatenate the root marketing url with 'xseries'  @marcotuts that's the desired destination, correct?